lib/game_teams_statistics_collection.rb

```diff
@@ -12,40 +12,7 @@ def initialize(game_teams_csv_location)
 
   def load_csv
     CSV.foreach(@game_teams_csv_location, :headers => true, :header_converters => :symbol) do |row|
-
-      game_id = row[:game_id]
-      team_id = row[:team_id]
-      hoa = row[:hoa]
-      result = row[:result]
-      settled_in = row[:settled_in]
-      head_coach = row[:head_coach]
-      goals = row[:goals]
-      shots = row[:shots]
-      tackles = row[:tackles]
-      pim = row[:pim]
-      powerplayopportunities = row[:powerplayopportunities]
-      powerplaygoals = row[:powerplaygoals]
-      faceoffwinpercentage = row[:faceoffwinpercentage]
-      giveaways = row[:giveaways]
-      takeaways = row[:takeaways]
-
-      @collection << GameTeams.new({
-        :game_id => game_id,
-        :team_id => team_id,
-        :hoa => hoa,
-        :result => result,
-        :settled_in => settled_in,
-        :head_coach => head_coach,
-        :goals => goals,
-        :shots => shots,
-        :tackles => tackles,
-        :pim => pim,
-        :powerplayopportunities => powerplayopportunities,
-        :powerplaygoals => powerplaygoals,
-        :faceoffwinpercentage => faceoffwinpercentage,
-        :giveaways => giveaways,
-        :takeaways => takeaways
-        })
+      @collection << GameTeams.new(row)
```

Comment:
You can just pass the row object as an argument -- if the GameTeams class already knows how to parse them, it doesn't have to be done here.